app/views/restaurants/show.html.erb

```diff
@@ -6,6 +6,7 @@
   <div>
     <h1> <%= @restaurant.name %></h1>
     <p> <%= @restaurant.address %></p>
+    <p> <%= @restaurant.description %></p>
```

Comment:
pls delete this line otherwise we have the description twice

Reply:
There is view conflicts on show – let go though them together when you are available?